mod1/src/file-5.xoo

```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:19188/organizations/org4/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:19188/project/issues?id=sample&branch=feature5&issues=AWEt9T7PFJlJ4KJ2TUs4&open=AWEt9T7PFJlJ4KJ2TUs4)

[ISSUE_KEY]: AWEt9T7PFJlJ4KJ2TUs4